app/components/form_components/autocomplete/sort/clean.js

```diff
@@ -1,6 +1,12 @@
-const clean = (word) => word.trim()
+const stopWords = ['the', 'of', 'in', 'and', 'at', '&']
+
+const removeStopWords = (text) => {
```

Comment:
I spent a bit of time thinking about where this feature should live. I gravitated towards the`clean` function because it's role related to removing stop words and since the code is relatively small, it was easy to tack on.

Reply:
Makes sense to me as fundamentally we're cleaning the stuff.

Stop words are a potential candidate for being configurable which makes them a _bit_ different too though.

Reply:
agreed! makes sense to me too 👍